app/uk/gov/hmrc/partnershipidentificationfrontend/forms/CaptureSautrForm.scala

```diff
@@ -22,7 +22,7 @@ import play.api.data.Forms._
 import scala.util.matching.Regex
 
 object CaptureSautrForm {
-  val sautrErrorKey: String = "sa-utr.error"
+  val sautrErrorKey: String = "capture-sa-utr.error"
   val sautrRegex: Regex = "[0-9]{10}".r
```

Comment:
Probably not worth changing, but this could also be done with a `sautr => sautr.forall(_.isDigit) && sautr.length == 10`

Not sure if that's a little clearer than the regex.